webpack.config.js

```diff
@@ -185,7 +185,7 @@ module.exports = function (env = {}) {
       },
     },
     // See https://webpack.js.org/configuration/devtool/
-    devtool: env.production ? "source-map" : "eval-source-map",
+    devtool: "source-map",
```

Comment:
The problem with this change is that according to the docs, `"eval-source-map"` is faster but should only be used in production (as we did before). Thus, keeping these changes might slow down our frontend rebuilding time upon changes to the frontend code in dev mode.